ambry-api/src/main/java/com/github/ambry/quota/storage/StorageQuotaEnforcer.java

```diff
@@ -65,6 +69,15 @@
    */
   boolean shouldThrottle(short accountId, short containerId, QuotaOperation op, long size);
```

Comment:
Do we need both the `shouldThrottle` methods in the interface. Maybe we can say this shouldThrottle is an implementation detail, and keep only one?

Reply:
updated.

Reply:
I see this is changed to `boolean shouldThrottle(RestRequest restRequest);`. This means ,on backend, we need to construct `RestRequest` somewhere?

Reply:
we don't need to, the storage quota service is only working in frontend.